README.md

```diff
@@ -40,37 +106,29 @@ dependency can be provided two ways:
 If you do not wish to build and run the tests, add `-DBENCHMARK_ENABLE_GTEST_TESTS=OFF`
 to `CMAKE_ARGS`.
 
+### Debug vs Release
 
-## Installation Guide
-
-For Ubuntu and Debian Based System
-
-First make sure you have git and cmake installed (If not please install them)
+By default, benchmark builds as a debug library. You will see a warning in the
+output when this is the case. To build it as a release library instead, use:
 
 ```
-sudo apt-get install git cmake
+cmake -DCMAKE_BUILD_TYPE=Release
 ```
 
-Now, let's clone the repository and build it
+To enable link-time optimisation, use
```

Comment:
Passing-by remark: unless LTO will be used when linking the library into the actual benchmark (a binary with `main()`), this is not quite useful, and maybe will even not work at all.

Reply:
Inclined to leave this for a separate update since it was present previously and I'm not in a position to verify this now.

Reply:
could one of you open an issue to capture this so we don't forget to look into it?

Reply:
See #805